buildpack/testdata/buildpack/by-id/B/v1/bin/build

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -euo pipefail
```

Comment:
I agree with you and debated symlinks initially, and have been debating them some more since you left this comment :) . I'm still of the opinion that symlinks are a little like regular expressions in that I've usually seen them solve a problem short-term by creating a different problem that happens later. It might even just be the confusion someone has when they think they're only editing the file for a single test-case but then all the other ones start to fail?   

but idk, it's very true that at this time, these tests look like i'm trying to maximize my LOC metrics 😁 - they're identical, redundant, _and_ the contents don't even really matter for the purposes of the tests.

Reply:
on further consideration, it looks like those files are entirely unnecessary, so i just deleted them :)